### DISCUSSION TITLE
Why is there a Limit on 32 Lists?

### DISCUSSION THREAD

**Discussion by kmf** [community/community#113251] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I love them lists.

**Comment by Mrxx99** [comment]
Yes, please exceed the limit, I love organizing my stars in lists, but now I have a problem because of that arbitrary limit..

**Reply by kaa123163** [reply]
http://localhost:8080/api/

**Reply by MXzander** [reply]
- [ ]

**Reply by Nico7777777** [reply]
They will probably do that in an immersive update to enhance the user accessibility, however to scale it up for hundreds of thousands of people wouldn't be that easy...

**Comment by meduzen** [comment]
I’m here for the same reason. I already reached the limit and I would like to have more to organize my [3K stars](https://github.com/meduzen?tab=stars).

**Comment by firmai** [comment]
This gives me 30% of my happiness on GitHub, why limit my job to 32 lists? It took me some time to write up, but I propose the following:

```
def get_starred_lists_limit():
   """
   Returns the maximum number of starred lists a user can have.
   """
   return 324 
```

**Reply by edrjnjdeuh73** [reply]
Thx.
Ad astra;,

**Reply by lfputnam** [reply]
[](url)

**Comment by lfputnam** [comment]
> This gives me 30% of my happiness on GitHub, why limit my job to 32 lists? It took me some time to write up, but I propose the following:
> 
> ```
> def get_starred_lists_limit():
>    """
>    Returns the maximum number of starred lists a user can have.
>    """
>    return 324 
> ```

**Reply by amiralighass** [reply]
Nic

**Reply by amiralighass** [reply]
Nice

**Reply by lfputnam** [reply]
```

```

**Comment by Jak12-3** [comment]
If a limit is required why is it not easy on android to see where they can be edited and consolidated?

**Comment by amiralighass** [comment]
Nice

**Reply by 0768993338** [reply]
We're There's A WIll There's A Way👃

**Comment by fabiodr** [comment]
More lists please, 100 or even 1000, to give a chance of organizing the mess.

**Reply by Matthewdavies1984** [reply]
07944851996

**Comment by Sashasissy** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I love them lists. 

OK

**Comment by swanepoelderreck** [comment]
🇹 🇭 🇮 🇸  🇮 🇸  🇹 🇭 🇪  🇳 🇪 🇼 
🇫 🇺 🇹 🇺 🇷 🇪 !

**Comment by rszymula** [comment]
Can we PLEASE get unlimited?

Lists were a AMAZING feature added, and we cannot get enough!

Please at 32 its still too short, unlimited would be great, but 128 should be enough!
So much easier to group repos that I want to reference later with LISTS!

**Comment by rszymula** [comment]
Ill pay extra $ to get more space in lists. If you really need justification for it, make it unlimited or 256 max for PAYING MEMBERS ONLY

**Comment by Giusantoos** [comment]
Cavaleirosombrio

**Comment by MXzander** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I love them lists. 

****

**Comment by likhitsomana** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I love them lists. 
> Author Fixture's Updates E-mailed.

**Comment by Milanhe92** [comment]
Milan je u tome što je u pitanju.

**Comment by Johngitah** [comment]
32

**Comment by DivitMittal** [comment]
+1

**Comment by sokai** [comment]
FYI: this discussion _seems_ about the **question, why** there is a/the limit, only …
If you wanna support a **FR to raise the limit**, you _maybe should react_ (with upvotes and 👍️) the (_initial_) [\[Feature Request\] Please Allow More Than 32 Lists! 🙏 #46887](https://github.com/orgs/community/discussions/46887)
KR w/ _hopes_
